x-pack/plugins/ml/public/application/routing/routes/memory_usage.tsx

```diff
@@ -35,13 +37,15 @@ export const nodesListRouteFactory = (
   enableDatePicker: true,
 });
 
-const PageWrapper: FC<PageProps> = ({ location, deps }) => {
-  const { context } = useResolver(
-    undefined,
-    undefined,
-    deps.config,
-    deps.dataViewsContract,
-    basicResolvers(deps)
+const PageWrapper: FC = () => {
+  const { context } = useRouteResolver(
+    'full',
+    ['canGetJobs', 'canGetDataFrameAnalytics', 'canGetTrainedModels'],
+    {
+      getMlNodeCount,
```

Comment:
Can we use basicResolvers() here?

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75